Sim_Harness_GUI/obj/Debug/Sim_Harness_GUI.gtk-gui.gui.stetic

```diff
@@ -7,22 +7,22 @@
     <widget-library name="glade-sharp, Version=2.12.0.0, Culture=neutral, PublicKeyToken=35e10195dab3c99f" />
```

Comment:
Sorry for barging in - I just wanted to make sure my attempts to help weren't lost to the ether. If you need any help, I'm happy to help explain things or offer helpful tricks, just let me know.